assets/src/block-templates/save.js

```diff
@@ -0,0 +1,6 @@
+const el = wp.element.createElement;
```

Comment:
Same suggestion as [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/929/files#r1011696458). Again, this is not required to change it 🙏